.editorconfig

```diff
@@ -6,6 +6,7 @@ indent_size = 2
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+end_of_line = lf
```

Comment:
We don't have to git track this file. Add this pattern to the .gitignore

Reply:
Actually this file has always been tracked.  I think you might be mixing it up with another file